OrbitUIProtos/capture_data.proto

```diff
@@ -4,7 +4,7 @@
 
 syntax = "proto3";
 
-package protos;
+package orbit_ui_protos;
```

Comment:
Soo, maybe this was discussed offline, but I'm not a big fun of this namespace name. "orbit" feels redundant and "ui" not too appropriate. Maybe "client_protos"?
Same for the module/directory name: I guess we want to keep "Orbit" for consistency with the other modules there, but maybe "OrbitClientProtos"?

Reply:
I don't remember if we agree on the certain name, so we can discuss it now. @dimitry- @antonrohr @beckerhe any other suggestions?

Reply:
we haven't we just discussed separating it from current protos - client sounds good to me, but I do not have strong feelings one way or the other.

Reply:
Ok, I'm not against OrbitClientProtos, and as no one objects, I renamed it to OrbitClientProtos.